File: DealApp.API/Models/Project.cs
using System;

namespace DealApp.API.Models
{
    public class Project
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Prifix { get; set; }
        public int? CityId { get; set; }
        public bool? IsActive { get; set; }
        public bool? IsDeleted { get; set; }
        public string CreatedBy { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string ModifyBy { get; set; }
        public DateTime? ModifyDate { get; set; }
        public string Mobile1 { get; set; }
        public string Mobile2 { get; set; }
        public string Address { get; set; }
        public string Email { get; set; }
        public string Website { get; set; }

        public virtual City City { get; set; }
    }
}

File: DealApp-SPA/src/app/_models/project.ts
export interface Project {
    id: number;
    name: string;
    prefix: string;
    cityId?: number;
    isActive?: boolean;
    isDeleted?: boolean;
    createdBy?: string;
    createdDate?: Date;
    modifiedBy?: string;
    modifiedDate?: Date;
    mobile1: string;
    mobile2: string;
    address: string;
    email: string;
    website: string;
}
